application/src/main/kotlin/net/corda/v5/application/crypto/SigningService.kt

```diff
@@ -36,4 +36,7 @@ interface SigningService {
      */
     @Suspendable
     fun sign(bytes: ByteArray, publicKey: PublicKey, signatureSpec: SignatureSpec): DigitalSignature.WithKey
+
+    @Suspendable
+    fun getMyKeys(keys: Set<PublicKey>): Set<PublicKey>
```

Comment:
This will be changed to a `Map<PublicKey, PublicKey>` mapping the input keys to the ones found to be "mine".